utils/generate_path_excludes.sh

```diff
@@ -0,0 +1,212 @@
+#!/bin/bash
+
+set -e
+shopt -s extglob
+
+# Dynamic Excludes
+#
+# This script generates two lists of files to exclude; one for the source, and one for the remote.
+#
+# Notes about excluded files:
+#
+#   Version Control
+#   NOTE:
+#     WP Engine does not support server side versioning so hosting any version control
+#     on the server would not be advantageous.
+#
+#   WordPress specific files
+#   NOTE:
+#     These files are excluded from the deploy so as to prevent unwanted errors from occurring,
+#     such as accidentally deploying a local version of wp-config.php or accidentally deleting
+#     wp-content/uploads/ if a --delete flag is passed while deploying root. Most paths here
+#     are ignored in the WPE sample .gitignore per best practice.
+#
+#   WP Engine specific files
+#   NOTE:
+#     These files are specific to running a WordPress site at WP Engine and would
+#     likely result in a broken production site if modified in production (in
+#     fact, permissions would prevent modification for many of these files). While
+#     some of these files (such as those in /_wpeprivate) would be extremely large
+#     and completely useless in the context of local WordPress development, others
+#     (such as some of the WP Engine managed plugins) might be useful in rare
+#     circumstances to have as a reference for debugging purposes.
+
+# Get the directory of the current script
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+# Source the print_mount_paths.sh file relative to the current script's location
+source "${SCRIPT_DIR}/print_mount_paths.sh"
+
+# Determine the source paths to exclude from the deployment
+determine_source_exclude_paths() {
+  local base_path
+  local mu_dir_path
```

Comment:
Using empty values here (instead of `/wp-content/` and `/wp-content/mu-plugins/`) might be a risk if some situation we've missed is not parsed on the logic below.

Is it better to always have an excludes path (and therefore always list the paths in the arrays from `print_dynamic_excludes` below)?